packages/generic/backend/src/lib/connector/buildConnector.ts

```diff
@@ -367,12 +367,14 @@ function buildConnector(connectorConfig: connectorConfigType) {
     return importedDocuments.length;
   }
 
-  async function importDocumentsSince({
+  async function importDocumentsSinceOrBetween({
```

Comment:
Tu peux mettre a jour les messages de log avec le nouveau nom de la fonction + logger le `to`